lib/render/editor/editor.css

```diff
@@ -62,3 +62,9 @@
 .gbs_gh {
   outline: 1px solid #cc0000 !important;
 }
+
+@-moz-document url-prefix() {
+  .mu-kids-exercise gs-board table.gbs_board {
+    position: absolute;
```

Comment:
Just thinking... how does this work? if we make the board's position `absolute` (only on Firefox), wouldn't it be floating over the blockly canvas?

Reply:
This is required because transform: scale doesn't work well on firefox and it is shifted upwards in the layout